samples/tests/create_dataset_video_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from uuid import uuid4
+
+import pytest
+import os
+
+from samples import create_dataset_video_sample
+from samples import delete_dataset_sample
+
+
+PROJECT_ID = os.getenv("BUILD_SPECIFIC_GCLOUD_PROJECT")
+VIDEO_METADATA_SCHEMA_URI = (
+    "gs://google-cloud-aiplatform/schema/dataset/metadata/video_1.0.0.yaml"
+)
+
+
+@pytest.fixture
+def shared_state():
+    state = {}
+    yield state
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
If you are defining the same fixture in different files, consider using `conftest.py` where you can define common fixture which are automatically usable in your tests.

Reply:
Great point - I've captured it as issue https://github.com/googleapis/python-aiplatform/issues/36 and self-assigned this as a todo.